elements/pfe-content-set/demo/index.html

```diff
@@ -1,328 +1,385 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <title>PatternFly Element | pfe-content-set Demo</title>
-    <link rel="stylesheet" href="https://overpass-30e2.kxcdn.com/overpass.css" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1" />
-    <title>PatternFly Element | pfe-content-set Demo</title>
-
-    <!-- Stylesheets for testing light DOM styles.
+
+<head>
+  <meta charset="utf-8">
+  <title>PatternFly Element | pfe-content-set Demo</title>
+  <link rel="stylesheet" href="https://overpass-30e2.kxcdn.com/overpass.css" />
```

Comment:
Do we still need to load overpass?

Reply:
Need to? Probably not. Though it looks nicer.. any reason why we shouldnt?

Reply:
Oh I thought we had switched the font away from overpass.

Reply:
ah, fair. switched to redhatfont